### ISSUE TITLE
Gap Fill Exercise

### ISSUE THREAD

**Issue by thexerteproject** [thexerteproject/xerteonlinetoolkits#372]
There are problems with the gap fill exercise after the v3 upgrade: mark word includes an additional space at the end of the selected word, consequently in text entry variants you need to also type the additional space to get a result.

There are problems with the layout, with marked words appearing on a line below the preceding text.

I can replicate this on our install, and on my xampp localhost.

**Comment by thexerteproject** [comment]
Regarding layout, I have added some code to strip out all <p> tags and to replace </p> with <br/><br/>. This fixes the layout issue. Not sure if this is a long term fix, but AFAICT, the editor inserts the initial <p> tags and adds further <p> tags if the user presses enter (expected), so it probably needs handling at runtime?

**Comment by ronm123** [comment]
@FayCross this might be a different issue but I was just working with a copy of the page types LO which obviously includes a gap fill page but wasn't displaying that page. This was using the develop code and I noticed there is some left over comments/changes which when removed resolves the issue. See screenshot below:
![image](https://cloud.githubusercontent.com/assets/5167146/10434078/25fecd9c-710f-11e5-91e2-19966b34cd86.png)
Happy to commit that change e.g. removing the 3 highlighted lines but perhaps you need to check first.
Ron

**Comment by FayCross** [comment]
Those comments shouldn't be there - they have been added with this change by Julian (https://github.com/thexerteproject/xerteonlinetoolkits/commit/8ccaffabda9d897ee7eaea232f60d2fc69e34ff7#diff-6f32e305c307ba2bbe268bdbe3cb69c3) Is this something odd that's happened with your copy of the repository Julian @thexerteproject?

I've just compared the gapfill model before that commit and after & I think the code below the mine section needs to be removed. I'll try to sort it out now.
